sources/packages/web/src/components/common/students/StudentProfile.vue

```diff
@@ -109,27 +113,39 @@ interface SharedStudentProfile extends Omit<StudentProfile, "sin"> {
 }
 
 export default defineComponent({
+  components: { DisabilityStatusUpdateTileValue },
   props: {
     studentId: {
       type: Number,
       required: true,
     },
+    allowDisabilityStatusUpdate: {
+      type: Boolean,
+      required: false,
+      default: false,
+    },
   },
   setup(props) {
     const studentDetail = ref({} as SharedStudentProfile);
     const address = ref({} as AddressAPIOutDTO);
-    const { sinDisplayFormat, emptyStringFiller } = useFormatters();
-    onMounted(async () => {
+    const { sinDisplayFormat, emptyStringFiller, disabilityStatusToDisplay } =
```

Comment:
Is the `disabilityStatusToDisplay` still needed in this component?
I believe that the component `disability-status-update-tile-value` is doing the translation also, right?